Pipfile

```diff
@@ -19,6 +19,7 @@ PyYAML = "==5.3.1"
 semver = "==2.10.2"
 jsonschema = "==3.2.0"
 pygithub = "==1.51"
+pybars3 = "*"
```

Comment:
I would freeze the version here
Don't forget to update again `Pipfile.lock` after changing this